app/javascript/dashboard/store/modules/helpCenterPortals/getters.js

```diff
@@ -0,0 +1,27 @@
+export const getters = {
+  uiFlagsIn: _state => helpCenterId => {
```

Comment:
Please use state. There is no need to use `_state`. `_` are generally used for private methods in js.

Reply:
Please update the names in the entire file.